tailwind.config.js

```diff
@@ -1,26 +1,173 @@
+                900: '#014737',
+            },
+            teal: {
+                50: '#edfafa',
+                100: '#d5f5f6',
+                200: '#afecef',
+                300: '#7edce2',
+                400: '#16bdca',
+                500: '#0694a2',
+                600: '#047481',
+                700: '#036672',
+                800: '#05505c',
+                900: '#014451',
+            },
+            blue: {
+                50: '#ebf5ff',
+                100: '#e1effe',
+                200: '#c3ddfd',
+                300: '#a4cafe',
+                400: '#76a9fa',
+                500: '#3f83f8',
+                600: '#1c64f2',
+                700: '#1a56db',
+                800: '#1e429f',
+                900: '#233876',
+            },
+            indigo: {
+                50: '#f0f5ff',
+                100: '#e5edff',
+                200: '#cddbfe',
+                300: '#b4c6fc',
+                400: '#8da2fb',
+                500: '#6875f5',
+                600: '#5850ec',
+                700: '#5145cd',
+                800: '#42389d',
+                900: '#362f78',
             },
+            purple: {
+                50: '#f6f5ff',
+                100: '#edebfe',
+                200: '#dcd7fe',
+                300: '#cabffd',
+                400: '#ac94fa',
+                500: '#9061f9',
+                600: '#7e3af2',
+                700: '#6c2bd9',
+                800: '#5521b5',
+                900: '#4a1d96',
+            },
+            pink: {
+                50: '#fdf2f8',
+                100: '#fce8f3',
+                200: '#fad1e8',
+                300: '#f8b4d9',
+                400: '#f17eb8',
+                500: '#e74694',
+                600: '#d61f69',
+                700: '#bf125d',
+                800: '#99154b',
+                900: '#751a3d',
+            },
+            lio: {
+                100: '#d1f2eb',
+                200: '#a3e4d7',
+                300: '#74d7c4',
+                400: '#46c9b0',
+                500: '#18bc9c',
+                600: '#15a589',
+                700: '#0e715e',
+                800: '#0a4b3e',
+                900: '#05261f',
+            },
+        },
+        fontSize: {
+            xs: '0.75rem',
+            sm: '0.875rem',
+            base: '1rem',
+            lg: '1.125rem',
+            xl: '1.25rem',
+            '2xl': '1.5rem',
+            '3xl': '1.875rem',
+            '4xl': '2.25rem',
+            '5xl': '3rem',
+            '6xl': '4rem',
+        },
+        extend: {
             fontFamily: {
                 sans: ['Inter var', ...defaultTheme.fontFamily.sans],
             },
         },
     },
-    variants: {},
-    plugins: [require('@tailwindcss/ui')],
+    variants: {
+        extend: {
+            ringColor: ['hover', 'active'],
+        },
+    },
+    plugins: [require('@tailwindcss/forms'), require('@tailwindcss/typography'), require('@tailwindcss/aspect-ratio')],
 };
```

Comment:
Let's update this one so it only contains the bare minimum we need: https://github.com/laravelio/paste.laravel.io/blob/main/tailwind.config.js